examples/bobandlynn.jsonld

```diff
@@ -0,0 +1,144 @@
+{
+    "@context": {
```

Comment:
should this reference `context.jsonld`, as in [elf's profile](https://github.com/valueflows/agent/blob/master/examples/elf-pavlik.jsonld#L3)?